docs/DocGenerator/Config.py

```diff
@@ -10,6 +10,8 @@ def __init__(self):
         self.documentation_path = ".."
```

Comment:
If you want to keep this file (with capital C) remove the other one (config.py)

Reply:
Fixed!